### PR TITLE
portfolio: init at 0.46.3

### LINKED ISSUE #35267: Packaging request: Portfolio Performance
Packaging request for https://github.com/buchen/portfolio

Feel free to comment here if you start working on it.

### PR DESCRIPTION
**Pull request by elohmeier**
###### Motivation for this change
Portfolio Performance package not available in nixpkgs.
Fixes #35267.

###### Things done
Packaged Portfolio Performance.

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by elohmeier** [comment]
Updated to the latest version (which dropped i686 support) and setup the webbrowser dependency for the holding diagram. Thanks @0xbe7a for your help :-)

**Comment by elohmeier** [comment]
Rebased onto latest master & configured generic jre.

**Comment by elohmeier** [comment]
@oyren @0xbe7a could you approve this PR? I think that would help getting it merged.

**Comment by elohmeier** [comment]
Updated to the latest version 0.46.3.

**Review by 0xbe7a — APPROVED** [review]

File: pkgs/applications/office/portfolio/default.nix [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+, glibc
+, gcc-unwrapped
+, autoPatchelfHook
+, fetchurl
+, libsecret
+, jre
+, makeWrapper
+, makeDesktopItem
+, webkitgtk
+, gtk3
```
Comment (oyren) [inline pkgs/applications/office/portfolio/default.nix:11]:
```suggestion
, webkitgtk
, wrapGAppsHook
, gtk3
```

File: pkgs/applications/office/portfolio/default.nix [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+  src = fetchurl {
+    url = "https://github.com/buchen/portfolio/releases/download/${version}/PortfolioPerformance-${version}-linux.gtk.x86_64.tar.gz";
+    sha256 = "0263v0ymqmf7kcwys4smwx1p1gh8d20dh10canjfqgl4c5lfx966";
+  };
+
+  nativeBuildInputs = [
+    autoPatchelfHook
+    makeWrapper
+  ];
```
Comment (oyren) [inline pkgs/applications/office/portfolio/default.nix:37]:
```suggestion
  nativeBuildInputs = [
    autoPatchelfHook
    makeWrapper
    wrapGAppsHook
  ];
```

**Review by oyren — CHANGES_REQUESTED** [review]
Hey with KDE I had the following error when opening a file, which led to a crash of the application:
`(Portfolio Performance:25632): GLib-GIO-ERROR **:  No GSettings schemas are installed on the system
`
With the suggested changes the application ran without problems except for some warnings like:
```
(Portfolio Performance:26694): dconf-WARNING **:  failed to commit changes to dconf: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: The name ca.desrt.dconf was not provided by any .service files

(Portfolio Performance:26694): Gtk-WARNING **: Negative content height -7 (allocation 3, extents 5x5) while allocating gadget (node scrollbar, owner GtkScrollbar)

(Portfolio Performance:26694): Gtk-CRITICAL **:  gtk_box_gadget_distribute: assertion 'size >= 0' failed in GtkScrollbar

```

**Comment by elohmeier** [comment]
@oyren Thanks for your test and review! I've updated the PR and included your suggested changes.

**Review by oyren — APPROVED** [review]
Thanks, @elohmeier. It works fine now.

**Pull request merged**